components/brave_wallet/browser/brave_wallet_constants.cc

```diff
@@ -99,4 +100,35 @@ const base::flat_map<std::string, std::string>& GetAnkrBlockchains() {
 
   return *blockchains;
 }
+
+const std::string GetWalletFrameSrcCSP() {
```

Comment:
Could we have comments above these two functions to document what string literal would be return (for each platform if needed)?